rules/docker_config.bzl

```diff
@@ -203,11 +204,11 @@ def _docker_toolchain_autoconfig_impl(ctx):
 
     # Command to copy produced toolchain configs to a tar at the root
     # of the container.
-    copy_cmd = []
+    copy_cmd = ["mkdir " + output_dir]
     for config_repo in ctx.attr.config_repos:
         src_dir = "$(bazel info output_base)/" + _EXTERNAL_FOLDER_PREFIX + config_repo
-        copy_cmd.append("cp -dr " + src_dir + " " + "/")
-    copy_cmd.append("tar -cf /" + outputs_tar + " /" + " /".join(ctx.attr.config_repos))
+        copy_cmd.append("cp -dr " + src_dir + " " + output_dir)
+    copy_cmd.append("tar -cf /" + outputs_tar + " -C " + output_dir + " . ")
```

Comment:
Will need trailing slash here after output_dir if you accept my other suggestion

Reply:
done